<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="4" failed="1" ignored="0" total="9" passed="4">
  <reporter-output>
    <line>
      <![CDATA[Internet Explorer Browser Open Sucessfully]]>
    </line>
    <line>
      <![CDATA[Windos maximize Sucessfully]]>
    </line>
    <line>
      <![CDATA[Testing for Reporter.log use it only use in TESTNG ]]>
    </line>
    <line>
      <![CDATA[Internet Explorer Browser Open Sucessfully]]>
    </line>
    <line>
      <![CDATA[Windos maximize Sucessfully]]>
    </line>
    <line>
      <![CDATA[Internet Explorer Browser Open Sucessfully]]>
    </line>
    <line>
      <![CDATA[Windos maximize Sucessfully]]>
    </line>
    <line>
      <![CDATA[Internet Explorer Browser Open Sucessfully]]>
    </line>
    <line>
      <![CDATA[Windos maximize Sucessfully]]>
    </line>
    <line>
      <![CDATA[Internet Explorer Browser Open Sucessfully]]>
    </line>
    <line>
      <![CDATA[Windos maximize Sucessfully]]>
    </line>
    <line>
      <![CDATA[Internet Explorer Browser Open Sucessfully]]>
    </line>
    <line>
      <![CDATA[Windos maximize Sucessfully]]>
    </line>
    <line>
      <![CDATA[Internet Explorer Browser Open Sucessfully]]>
    </line>
    <line>
      <![CDATA[Windos maximize Sucessfully]]>
    </line>
    <line>
      <![CDATA[Internet Explorer Browser Open Sucessfully]]>
    </line>
    <line>
      <![CDATA[Windos maximize Sucessfully]]>
    </line>
    <line>
      <![CDATA[Internet Explorer Browser Open Sucessfully]]>
    </line>
    <line>
      <![CDATA[Windos maximize Sucessfully]]>
    </line>
  </reporter-output>
  <suite name="Medisolv Encor-E RetryFailedTests" duration-ms="71985" started-at="2019-11-07T06:34:04Z" finished-at="2019-11-07T06:35:16Z">
    <groups>
    </groups>
    <test name="Encor-E Application Test Cases" duration-ms="71985" started-at="2019-11-07T06:34:04Z" finished-at="2019-11-07T06:35:16Z">
      <class name="com.encore.testcases.ReportingPage_TestCases">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="setUp" is-config="true" duration-ms="16801" started-at="2019-11-07T12:04:04Z" finished-at="2019-11-07T12:04:20Z">
          <reporter-output>
            <line>
              <![CDATA[Internet Explorer Browser Open Sucessfully]]>
            </line>
            <line>
              <![CDATA[Windos maximize Sucessfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="VerifyReportingPageTitle()[pri:1, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="VerifyReportingPageTitle" duration-ms="24" started-at="2019-11-07T12:04:20Z" finished-at="2019-11-07T12:04:20Z">
          <reporter-output>
            <line>
              <![CDATA[Testing for Reporter.log use it only use in TESTNG ]]>
            </line>
          </reporter-output>
        </test-method> <!-- VerifyReportingPageTitle -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="tearDown" is-config="true" duration-ms="531" started-at="2019-11-07T12:04:20Z" finished-at="2019-11-07T12:04:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="setUp" is-config="true" duration-ms="4918" started-at="2019-11-07T12:04:21Z" finished-at="2019-11-07T12:04:26Z">
          <reporter-output>
            <line>
              <![CDATA[Internet Explorer Browser Open Sucessfully]]>
            </line>
            <line>
              <![CDATA[Windos maximize Sucessfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="ClickOn_Reporting_Test()[pri:2, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="ClickOn_Reporting_Test" duration-ms="732" started-at="2019-11-07T12:04:26Z" finished-at="2019-11-07T12:04:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ClickOn_Reporting_Test -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="tearDown" is-config="true" duration-ms="534" started-at="2019-11-07T12:04:27Z" finished-at="2019-11-07T12:04:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="setUp" is-config="true" duration-ms="4941" started-at="2019-11-07T12:04:27Z" finished-at="2019-11-07T12:04:32Z">
          <reporter-output>
            <line>
              <![CDATA[Internet Explorer Browser Open Sucessfully]]>
            </line>
            <line>
              <![CDATA[Windos maximize Sucessfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="Display_All_ReportingMenu_Test()[pri:3, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="Display_All_ReportingMenu_Test" duration-ms="3049" started-at="2019-11-07T12:04:32Z" finished-at="2019-11-07T12:04:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Display_All_ReportingMenu_Test -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="tearDown" is-config="true" duration-ms="625" started-at="2019-11-07T12:04:35Z" finished-at="2019-11-07T12:04:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="setUp" is-config="true" duration-ms="4898" started-at="2019-11-07T12:04:36Z" finished-at="2019-11-07T12:04:41Z">
          <reporter-output>
            <line>
              <![CDATA[Internet Explorer Browser Open Sucessfully]]>
            </line>
            <line>
              <![CDATA[Windos maximize Sucessfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="Display_HospMenu_Test()[pri:4, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="Display_HospMenu_Test" duration-ms="537" started-at="2019-11-07T12:04:41Z" finished-at="2019-11-07T12:04:41Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Expected is : Hospital Measures: TestCase Failed !!! expected [Hospital Measures] but found [Hospital Measure]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Expected is : Hospital Measures: TestCase Failed !!! expected [Hospital Measures] but found [Hospital Measure]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at com.encore.testcases.ReportingPage_TestCases.Display_HospMenu_Test(ReportingPage_TestCases.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Display_HospMenu_Test -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="tearDown" is-config="true" duration-ms="530" started-at="2019-11-07T12:04:41Z" finished-at="2019-11-07T12:04:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="setUp" is-config="true" duration-ms="4904" started-at="2019-11-07T12:04:42Z" finished-at="2019-11-07T12:04:47Z">
          <reporter-output>
            <line>
              <![CDATA[Internet Explorer Browser Open Sucessfully]]>
            </line>
            <line>
              <![CDATA[Windos maximize Sucessfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="Display_HospMenu_Test()[pri:4, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="Display_HospMenu_Test" duration-ms="2307" started-at="2019-11-07T12:04:47Z" finished-at="2019-11-07T12:04:49Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Expected is : Hospital Measures: TestCase Failed !!! expected [Hospital Measures] but found [Hospital Measure]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Expected is : Hospital Measures: TestCase Failed !!! expected [Hospital Measures] but found [Hospital Measure]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at com.encore.testcases.ReportingPage_TestCases.Display_HospMenu_Test(ReportingPage_TestCases.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:998)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1200)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Display_HospMenu_Test -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="tearDown" is-config="true" duration-ms="648" started-at="2019-11-07T12:04:49Z" finished-at="2019-11-07T12:04:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="setUp" is-config="true" duration-ms="4958" started-at="2019-11-07T12:04:50Z" finished-at="2019-11-07T12:04:55Z">
          <reporter-output>
            <line>
              <![CDATA[Internet Explorer Browser Open Sucessfully]]>
            </line>
            <line>
              <![CDATA[Windos maximize Sucessfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="Display_HospMenu_Test()[pri:4, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="Display_HospMenu_Test" duration-ms="574" started-at="2019-11-07T12:04:55Z" finished-at="2019-11-07T12:04:55Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Expected is : Hospital Measures: TestCase Failed !!! expected [Hospital Measures] but found [Hospital Measure]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Expected is : Hospital Measures: TestCase Failed !!! expected [Hospital Measures] but found [Hospital Measure]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at com.encore.testcases.ReportingPage_TestCases.Display_HospMenu_Test(ReportingPage_TestCases.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:998)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1200)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Display_HospMenu_Test -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="tearDown" is-config="true" duration-ms="551" started-at="2019-11-07T12:04:55Z" finished-at="2019-11-07T12:04:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="setUp" is-config="true" duration-ms="5100" started-at="2019-11-07T12:04:56Z" finished-at="2019-11-07T12:05:01Z">
          <reporter-output>
            <line>
              <![CDATA[Internet Explorer Browser Open Sucessfully]]>
            </line>
            <line>
              <![CDATA[Windos maximize Sucessfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="Display_HospMenu_Test()[pri:4, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="Display_HospMenu_Test" duration-ms="746" started-at="2019-11-07T12:05:01Z" finished-at="2019-11-07T12:05:01Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Expected is : Hospital Measures: TestCase Failed !!! expected [Hospital Measures] but found [Hospital Measure]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Expected is : Hospital Measures: TestCase Failed !!! expected [Hospital Measures] but found [Hospital Measure]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at com.encore.testcases.ReportingPage_TestCases.Display_HospMenu_Test(ReportingPage_TestCases.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:998)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1200)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Display_HospMenu_Test -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="tearDown" is-config="true" duration-ms="651" started-at="2019-11-07T12:05:01Z" finished-at="2019-11-07T12:05:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="setUp" is-config="true" duration-ms="5287" started-at="2019-11-07T12:05:02Z" finished-at="2019-11-07T12:05:07Z">
          <reporter-output>
            <line>
              <![CDATA[Internet Explorer Browser Open Sucessfully]]>
            </line>
            <line>
              <![CDATA[Windos maximize Sucessfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="Display_HospMenu_Test()[pri:4, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="Display_HospMenu_Test" duration-ms="528" started-at="2019-11-07T12:05:07Z" finished-at="2019-11-07T12:05:08Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Expected is : Hospital Measures: TestCase Failed !!! expected [Hospital Measures] but found [Hospital Measure]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Expected is : Hospital Measures: TestCase Failed !!! expected [Hospital Measures] but found [Hospital Measure]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at com.encore.testcases.ReportingPage_TestCases.Display_HospMenu_Test(ReportingPage_TestCases.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:998)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1200)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Display_HospMenu_Test -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="tearDown" is-config="true" duration-ms="637" started-at="2019-11-07T12:05:08Z" finished-at="2019-11-07T12:05:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="setUp" is-config="true" duration-ms="5851" started-at="2019-11-07T12:05:09Z" finished-at="2019-11-07T12:05:14Z">
          <reporter-output>
            <line>
              <![CDATA[Internet Explorer Browser Open Sucessfully]]>
            </line>
            <line>
              <![CDATA[Windos maximize Sucessfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="Display_EH_MenuList_Test()[pri:5, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="Display_EH_MenuList_Test" duration-ms="443" started-at="2019-11-07T12:05:14Z" finished-at="2019-11-07T12:05:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Display_EH_MenuList_Test -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.encore.testcases.ReportingPage_TestCases@3f49dace]" name="tearDown" is-config="true" duration-ms="631" started-at="2019-11-07T12:05:15Z" finished-at="2019-11-07T12:05:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.encore.testcases.ReportingPage_TestCases -->
    </test> <!-- Encor-E Application Test Cases -->
  </suite> <!-- Medisolv Encor-E RetryFailedTests -->
</testng-results>
